internal/storage/runtime.go

```diff
@@ -258,6 +258,7 @@ func (r *runtimeService) createContainerOrPodSandbox(systemContext *types.System
 
 	coptions := storage.ContainerOptions{
 		LabelOpts: labelOptions,
+		Volatile:  true,
```

Comment:
Can we do this through a conf option? That way we can test and back out if needed easily.

Reply:
can't we pass it in storage_opts in storage.conf?

Reply:
No because, we only want it for certain volume mounts.

Reply:
what we could do is to have a global configuration `disable_volatile` that always disables it

Reply:
Would this be ignored if the caller asked for it?  IE Podman run --rm?

Reply:
yes, if the user disables completely support for "volatile" then it won't be used for any container.

PR: https://github.com/containers/storage/pull/903

I think it is just a last resort useful to CRI-O